deepfence_worker/tasks/reports/data.go

```diff
@@ -258,11 +275,13 @@ func getSecretData(ctx context.Context, params sdkUtils.ReportParams) (*Info[mod
 		params.Filters.SeverityOrCheckType, params.Filters.AdvancedReportFilters.Masked)
 
 	nodeWiseData := NodeWiseData[model.Secret]{
-		SeverityCount: make(map[string]map[string]int32),
-		ScanData:      make(map[string]ScanData[model.Secret]),
+		SeverityCount:         make(map[string]map[string]int32),
+		ScanData:              make(map[string]ScanData[model.Secret]),
+		OverallSeverityCounts: make(map[string]int32),
 	}
-
+	overallSeverityCounts := CalculateOverallSeverityCounts()
 	for _, s := range scans {
+		overallSeverityCounts = CalculateOverallSeverityCounts(overallSeverityCounts, s.SeverityCounts)
```

Comment:
That works but we create new maps every time we call `CalculateOverallSeverityCounts`.
Let's simplify by having 1 `overallSeverityCounts` declared in this scope & keep adding onto the same map

Reply:
I have addressed it by passing the `nodeWiseData` itself to the `CalculateOverallSeverityCounts` function."